content/writing/digital-approaches-for-social-change-organisations.mdx

```diff
@@ -0,0 +1,112 @@
+
+## Core themes
+
+We begun our design process by asking ourselves: how might we make JRF's extensive research into poverty more accessible to a broad audience? We looked at ways to enable anyone – whether that's campaigners, people with lived experience of poverty, or individuals wanting to take action – to find the most relevant information to them and (crucially) to use it in their organising efforts.
+
+Our initial wireframes explored a broad range of ideas, loosely grouped under the theme "data for action". We explored how to weave other voices – either grassroots campaigners or people with lived experience – into JRF's reports, how to contextualise datasets through postcode search, and how to slice up their research into easily shareable and exportable chunks.
+
+At the same time, we were conducting research interviews with organisers. From this, and the feedback we gathered in our show-and-tell sessions, the idea of relationship-building emerged as the most crucial question. JRF have already built relationships with grassroots organisations and individuals, but there's so much potential to deepen these relationships and form new ones.
+
+Based on this feedback, we refined our focus a little and we started exploring how to enable relational organising through JRF's website. We looked for ways to enable people to build relationships with each other directly, rather than always being mediated through JRF.  For example, JRF could provide an open directory of movement organisations in their network, spotlighting or platforming different groups whenever relevant. They could explore other digital spaces, like Slack, to allow people to meet each other, work together and cross-pollinate between disciplines.
+
+On reflection, our ideas here were somewhat atomised – sketches of minor interventions rather than a more holistic strategy. So we zoomed back out and asked ourselves: how might we create a digital product that is centred around building relationships, coalitions and bridges between different organisations.
+
+We prototyped a new topic overview page full of curated, accessible information and signposts towards opportunities for relationship-building. We envisioned this page to include:
+
+- A top-level explanation of the issue, to surface the most important learnings that JRF make through their research
+- Key policy recommendations distilled from a variety of reports
+- Contextualised data visualisations
+- An open directory and map of other organisations
+- Featured projects, to demonstrate how research is being used on the ground
+- Staff profiles, linked to an open booking system and accompanied by conversation prompts
```

Comment:
"The intention being to make the expertise of JRF porous to the wider movements, in both directions"

I'd be tempted to add a quck chaser of "why" to each of these.